charts/hocs-txa-document-extractor/values.yaml

```diff
@@ -0,0 +1,18 @@
+---
+jobs:
+  cs-ingests:
+    enabled: true
+    schedule: '5 8 * * *'
+    imageTag: 2.0.0
+  cs-deletes:
+    enabled: true
+    schedule: '5 8 * * *'
+    imageTag: 2.0.0
+  wcs-ingests:
+    enabled: true
+    schedule: '5 8 * * *'
+    imageTag: 2.0.0
+  wcs-deletes:
+    enabled: true
+    schedule: '5 8 * * *'
+    imageTag: 2.0.0'
```

Comment:
@walsterc L18 seems to have an erroneous `'` at the end

Reply:
Yep, sorry. Have created another PR to fix that https://github.com/UKHomeOffice/hocs-helm-charts/pull/133/files

That also fixes another typo I noticed but forgot to push before